File: list_coda_tables.py
import requests

CODA_TOKEN = "bba1cc5d-35d7-4772-a866-c95f2310d071"
DOC_ID = "NqBfudo5pw"

def list_tables():
    url = f"https://coda.io/apis/v1/docs/{DOC_ID}/tables"
    headers = {
        "Authorization": f"Bearer {CODA_TOKEN}"
    }
    response = requests.get(url, headers=headers)
    print(f"Status: {response.status_code}")
    if response.status_code == 200:
        tables = response.json().get('items', [])
        print("Tabelas encontradas:")
        for t in tables:
            print(f"- {t['name']} (ID: {t['id']})")
    else:
        print(response.text)

if __name__ == "__main__":
    list_tables()

File: list_coda_tables.js
const CODA_API_KEY = process.env.CODA_API_KEY;
const DOC_ID = "NqBfudo5pw";

async function listTables() {
    console.log(`Listing tables for doc: ${DOC_ID}`);
    const res = await fetch(`https://coda.io/apis/v1/docs/${DOC_ID}/tables`, {
        headers: { "Authorization": `Bearer ${CODA_API_KEY}` }
    });
    if (!res.ok) {
        console.error(`Error: ${res.status} ${await res.text()}`);
        return;
    }
    const data = await res.json();
    console.log(JSON.stringify(data.items.map(t => ({ name: t.name, id: t.id })), null, 2));
}

listTables();
